test/fixtures/datalinks-status-response.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "id": "123456",
+    "status": {
+      "status": "ONLINE",
+      "requestedAt": "2017-11-06T03:42:31.970Z",
+      "deployUserInfo": {
+        "firstName": "Davy",
+        "lastName": "Jones",
+        "email": "davy.jones@mail.com"
+      },
+      "version": "1.4.2",
+      "runtime": "flex-runtime-node8",
+      "deployment": {
+        "status": "COMPLETED",
+        "version": "1.4.2",
```

Comment:
It is good 'version' to be different from 'deployment.version' in order to validate that the CLI reads and displays the correct fields from MAPI